src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseDataFileService.cs

```diff
@@ -452,13 +449,19 @@ private async Task<DataFileInfo> GetFallbackDataFileInfo(Guid releaseId, File da
 
         private async Task<string> GetSubjectName(File file)
```

Comment:
There's one case

```
Name = dataFile.SubjectId.HasValue ? await GetSubjectName(dataFile) : blob.Name,
```

That can now just be `Name = await GetSubjectName(dataFile)`

and another case

```
Name = zipBlob.Name
```

that can be `Name = await GetSubjectName(dataFile)`